packages/docsearch-react/src/utils/buildInsightsClickParams.ts

```diff
@@ -0,0 +1,22 @@
+import type { AlgoliaInsightsHit } from '@algolia/autocomplete-core';
+import type {
+  InsightsParamsWithItems,
+  ClickedObjectIDsAfterSearchParams,
+} from '@algolia/autocomplete-plugin-algolia-insights';
+
+import type { InternalDocSearchHit } from '../types';
+
+export function buildInsightsClickParams(
```

Comment:
It looks like this function content can be inlined in `sendItemClickEvent`?

Reply:
Yeah I agree, originally I had the insights inlined to the click handler so this felt better then. Now that it's broken out this makes sense